### ISSUE TITLE
Initial guess is propagated using unscaled values

### ISSUE THREAD

**Issue by msparapa** [Rapid-Design-of-Systems-Laboratory/beluga#135]
Even when we use scaling, the initial guess is generated using unscaled values. These should at least be consistent on whether or not the `autoscale` keyword is `True` or `False` within `beluga.solve()`.

**Comment by SeanMatthewNolan** [comment]
Is this a problem? Maybe it is, but propagating equations forward has rarely if ever required scaling for me. Also, how does the solver know how to scale some values prior to the guess. For example, if you scale `m` by `x` but `x` has an initial value of 0, how will that be handled?

**Comment by msparapa** [comment]
Good point. I have a case where the scaling plays a huge role in how long the forward propagation takes. It pretty much gets stuck at the initial guess stage when not scaled. I'll need to think about this one some more.

**Comment by msparapa** [comment]
Closing this as something that won't be fixed since I've had more time to think about it. Since I've opened this issue, I've made a lot of the techniques we use more abstract and as a result, they're more interconnected. I.e. eventually a problem solved by direct methods can be used as an initial guess into indirect, or a problem solved by ICRM can be used directly as a guess in non-ICRM strats. In my case where forward propagation takes forever, I'm going to solve a simpler case then use that result as a guess into the more complicated one.
